### PR TITLE
Tests: Multiple test case port

### PR DESCRIPTION
**Pull request by aborah-sudo**
newusers creates users with negative UID and GID
groupmems -d does not work
Huge sparse files /var/log/lastlog and /var/log/faillog creating system problems Make sure chpasswd does not segfault under some conditions

### CODE REVIEW

File: multihost_test/Bugzillas/test_Regression.py [hunk 1]
```diff
@@ -266,3 +266,143 @@ def test_bz455603(self, multihost, create_backup):
...
+        client.run_command(f'groupmems -a {test_user} -g {test_group}')
+        client.run_command(f'grep "^{test_group}:" /etc/group')
+        client.run_command(f'grep "^{test_group}:" /etc/group | grep ":{test_user}$" >& /dev/null')
+        client.run_command(f'groupmems -d {test_user} -g {test_group}')
+        client.run_command(f'grep "^{test_group}:" /etc/group')
+        with pytest.raises(Exception):
+            client.run_command(f'grep "^{test_group}:" /etc/group | grep ":{test_user}$" >& /dev/null')
+        client.run_command(f'groupdel {test_group}')
+        client.run_command(f'userdel -r {test_user}')
+    @pytest.mark.tier1
```
Comment (ikerexxe) [inline multihost_test/Bugzillas/test_Regression.py:354]: Add a blank line

File: multihost_test/Bugzillas/test_Regression.py [hunk 2]
```diff
@@ -266,3 +266,143 @@ def test_bz455603(self, multihost, create_backup):
...
+          4. Should succeed
+          5. Should succeed
+          6. Should succeed
+          7. Should succeed
+          8. Should succeed
+          9. Should succeed
+          10. Should succeed
+        """
+        client = multihost.client[0]
+        score = 0
```
Comment (ikerexxe) [inline multihost_test/Bugzillas/test_Regression.py:339]: What's the purpose of this variable?

File: multihost_test/Bugzillas/test_Regression.py [hunk 3]
```diff
@@ -266,3 +266,143 @@ def test_bz455603(self, multihost, create_backup):
...
+        :id: 1b9a2ed0-f0c9-11ee-989d-845cf3eff344
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=247514
+        :steps:
+          1. Add a new user named "foo247514" and sets the password for the user with chpasswd.
+          2. Delete the user "foo247514" along with their home directory and mail spool.
+          3. Add the user "foo247514" again and set their password to "password" with chpasswd.
+        :expectedresults:
+          1. Should succeed
+          2. Should succeed
+          3. There should not be segfault
```
Comment (ikerexxe) [inline multihost_test/Bugzillas/test_Regression.py:401]: I'd change this by `Should succeed (no segfault)`

File: multihost_test/Bugzillas/test_Regression.py [hunk 4]
```diff
@@ -266,3 +266,142 @@ def test_bz455603(self, multihost, create_backup):
...
+    @pytest.mark.tier1
+    def test_bz455609(self, multihost, create_backup):
+        """
+        :title: groupmems -d does not work
+        :id: 0d62b45e-f0c9-11ee-b808-845cf3eff344
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=455609
+        :steps:
+          1. Add a test user to the system, and the score is incremented
+          2. Add a test group to the system, and the score is incremented
+          3. Add the test user to the test group, and the score is incremented.
```
Comment (ikerexxe) [inline multihost_test/Bugzillas/test_Regression.py:318]: You removed the `score` variable but it's still mentioned in the steps. Do you know its purpose? I'm curious to understand why the original engineer added this variable

Comment (aborah-sudo) [reply multihost_test/Bugzillas/test_Regression.py:318]: I have modified this test , we dont need score variable now . So some reason the  original engineer added this variable to match line of command executed successfully. But we do not need this any more .

**Review by ikerexxe — APPROVED** [review]
LGTM!

**Pull request merged**